library/Vanilla/Models/FsThemeProvider.php

```diff
@@ -54,6 +57,7 @@ public function __construct(
         $this->addonManager = $addonManager;
         $this->request = $request;
         $this->config = $config;
+        $this->themeOptionValue = $this->config->get('Garden.ThemeOptions.Styles.Value', null);
```

Comment:
Could we set some default value instead of `null`? Like: `''` or `[]`.
That would help to determine if property was set to empty or it was never set and because of that it is `null`.

Reply:
In the end it doesn't really matter, because there is a value in `config-defaults`, but I can do that.

Reply:
Done